### ISSUE TITLE
Client side JS rendering error re: browser-ponyfill.js does not provide an export named 'default'

### ISSUE THREAD

**Issue by JamesPHoughton** [empiricaly/empirica#449]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

After bumping to latest version, running the admin console is fine, but switching to participant view yields a JS error, and a blank page. (Nothing is rendered at all.)

### From chrome browser console:
```
Uncaught SyntaxError: The requested module '/node_modules/cross-fetch/dist/browser-ponyfill.js?v=c60c6b24' does not provide an export named 'default' (at graphql.ts:219:3)
```

### From firefox
``` 
Uncaught SyntaxError: ambiguous indirect export: default index.js:4:8
```
Which corresponds to:

![image](https://github.com/empiricaly/empirica/assets/4304478/2dce1879-ddef-4cee-a810-1a6a3d751fad)

### From safari:
<img width="643" alt="image" src="https://github.com/empiricaly/empirica/assets/4304478/ad45378f-120f-434e-8f52-37cf2f94c7ec">

### Empirica Version

```text
Version: v1.8.8
SHA:     c2176a6
Branch:  main
Time:    2023-12-03T08:21:53Z

Client:  1.8.8
Server:  1.8.8
```

### What OS are you seeing the problem on?

macOS

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by JamesPHoughton** [comment]
Looks like this is something other projects have encountered using graphql and `cross-fetch`: https://github.com/lquixada/cross-fetch/issues/120#issuecomment-1617172860

Some discussion in other projects about it being related to how the packages are linked?

**Comment by malsobay** [comment]
@JamesPHoughton thanks for the detailed report -- just chiming in to say I'm also facing this. happy to share any diagnostics that are helpful, but I see obvious bases are already covered.

**Comment by npaton** [comment]
Would mind trying to add the following to you client/package.json and then `npm i` and try again:

```json
  "overrides": {
    "@empirica/core": {
      "@empirica/tajriba": "1.5.2"
    }
  }
```

Thanks!

**Comment by JamesPHoughton** [comment]
That worked great - what is it doing?

**Comment by npaton** [comment]
This should be fixed in release 1.8.9.
